lib/core/include/irods/client_connection.hpp

```diff
@@ -47,34 +66,98 @@ namespace irods::experimental
         /// \param[in] _host     The host name of the iRODS server.
         /// \param[in] _port     The port to connect to.
         /// \param[in] _username The user to connect as.
-        /// \param[in] _zone     The zone managed by the iRODS server.
+        /// \param[in] _zone     The zone of the user.
         ///
         /// \since 4.2.9
-        client_connection(const std::string_view _host,
+        [[deprecated]] client_connection(const std::string_view _host,
```

Comment:
That's true. I only marked it deprecated because other projects we manage likely use these constructors, but as we already declared, anything within the experimental namespace can be torched at anytime.

Will think about this more.

Reply:
I think I'm going to remove these constructors.

Nice catch on the `experimental` namespace. It's working!